website/docs/guides/3.0-upgrade-guide.html.markdown

```diff
@@ -267,6 +267,18 @@ The deprecated field `management_group_id` will be removed in favour of the `man
 
 The deprecated field `cmk_enabled` will be removed since it no longer exists in the Azure API.
 
+### Data Source: `azurerm_sql_database`
+
+The data source `azurerm_sql_database` is deprecated in favour of the `azurerm_mssql_database` data source.
```

Comment:
I'm unsure whether this really needs to be added to the upgrade guide since it's just a deprecation and not a breaking change yet. Ultimately I suppose there isn't any harm in adding these but it does add quite a bit of size to an already long upgrade guide?

Reply:
Hi @stephybun, great point, for the sake of brevity and avoiding precedent, I think that's the right call. Have dropped that 👍